shared/macros-oval.jinja

```diff
@@ -17,8 +17,9 @@
     - multi_value (boolean): If set, it means that the parameter can accept multiple values and the expected value must be present in the current list of values.
     - missing_config_file_fail (boolean): If set, the check will fail if the configuration is not existent in the system.
     - section (String): If set, the parameter will be checked only within the given section defined by [section].
+    - quotes (String): If non-empty, one level of matching quotes is considered when checking the value. See comment of oval_line_in_file_state for more info.
 #}}
-{{%- macro oval_check_config_file(path='', prefix_regex='^[ \\t]*', parameter='', separator_regex='[ \\t]+', value='', missing_parameter_pass=false, application='', multi_value=false, missing_config_file_fail=false, section='') -%}}
+{{%- macro oval_check_config_file(path='', prefix_regex='^[ \\t]*', parameter='', separator_regex='[ \\t]+', value='', missing_parameter_pass=false, application='', multi_value=false, missing_config_file_fail=false, section='', quotes='') -%}}
```

Comment:
Do you think that we should extend the tests of this macro (`macro oval_check_config_file`) with the new parameter to make sure it's correctly propagated and not ignored?

Reply:
There is a list for quotes and no-quotes OVAL, is that what you meant, or do you still miss some other tests?

Reply:
You add a lot of tests for oval_check_shell_file, which is great. But, you modify also the parent macro oval_check_config_file, this addition is tested only indirectly. Do you think that we should add this new option also to test cases that call "oval_check_config_file"?